### PR TITLE
🚀 Use a multislot request for pubmine ads

### PR DESCRIPTION
**Pull request by shackijj**
At the moment each pubmine slot sends an ad request. This PR contains changes allowing sending one ad request for multiple slots.

### CODE REVIEW

**Comment by shackijj** [comment]
@lannka @zhouyx @dbspringer Hello! Could you please review this PR?

**Comment by dbspringer** [comment]
Thanks for the update, this looks good/tests well from Automattic's side 👍

**Comment by lannka** [comment]
@calebcordry could you help on this?

**Comment by calebcordry** [comment]
@shackijj Thanks for contributing! Will review today.

File: ads/pubmine.js [hunk 1]
```diff
@@ -20,46 +20,57 @@ const pubmineOptional = ['section', 'pt', 'ht'],
...
+/**
+ * @param {string} slotId
+ */
+function createSlot(slotId) {
+  const containerEl = document.getElementById('c');
+  const adSlot = document.createElement('div');
+  adSlot.setAttribute('id', slotId);
+  containerEl.appendChild(adSlot);
+}
 /**
```
Comment (calebcordry) [inline ads/pubmine.js]: nit: newline

**Comment by shackijj** [comment]
@calebcordry I've commited fixes covering your comments below. Unfortunately, it seems that there is an infrastructural problem on the CI server. Do you know how to restrart the job?

**Comment by calebcordry** [comment]
@shackijj I went ahead and restarted. I will try to keep an eye on this, but feel free to ping me if you see an update. Thanks.

**Comment by shackijj** [comment]
@calebcordry the e2e tests failed again.

**Comment by calebcordry** [comment]
This actually is the same test failing so it may not be a flake. I don't think it is related to your changes. Could you try rebasing and see if it helps?

**Comment by shackijj** [comment]
@calebcordry Hello! After rebasing all checks have passed. Thanks.

**Comment by calebcordry** [comment]
Awesome!

**Comment by calebcordry** [comment]
Thanks for contributing, and your patience with flaky tests :)

**Pull request merged**